app/config.js

```diff
@@ -7,6 +7,7 @@ const schema = Joi.object({
   port: Joi.number().default(3001),
   env: Joi.string().valid('development', 'test', 'production').default('development'),
   staticCacheTimeoutMillis: Joi.number().default(7 * 24 * 60 * 60 * 1000), // 1 day
+  publishPollingInterval: Joi.number().default(5000),
```

Comment:
What default value should this have?

Reply:
I think what you have is probably a good starting point so let's go with it for now.